### PR TITLE
Create mulamadhymakakarika_garfield-jay.md

### PR DESCRIPTION
**Pull request by JJM317**
I found this to be an odd one. I understand the 70 verses are selected from a revered monk, but as a document in the library it seems strange. Let's switch this over to Garfield's book which these translations are from?

I couldn't locate a link to this besides the drive. I left external an entry has to have one. Can you point me in the right direction?

Also, is United Kingdom the correct address for the oxford slug?

### CODE REVIEW

File: _content/excerpt/mulamadhymakakarika_garfield-jay.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: "Mulamadhymakakarika"
```
Comment (buddhist-uni) [inline _content/excerpt/mulamadhymakakarika_garfield-jay.md:2]: 🤔 In this case I'm inclined to add something to the title to indicate that this isn't the entire thing... What do you think? `"[Selected Verses from] Mulamadhyamakakarika"`?

File: _content/excerpt/mulamadhymakakarika_garfield-jay.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  - madhyamaka
+  - mahayana
+  - emptiness
+year: 1995 
+oclc: 32012411 
+publisher: oxford
+address: "United Kingdom"
+booktitle: "The fundamental wisdom of the middle way: Nāgārjuna's Mūlamadhyamakakārikā"
+---
+A translation of a select seventy verses from Nāgārjuna's Mūlamadhyamakakārikā.
```
Comment (buddhist-uni) [inline _content/excerpt/mulamadhymakakarika_garfield-jay.md:20]: This is good for a second paragraph, but this definitely deserves a pull quote. Perhaps,

```markdown
> By a misperception of emptiness  
A person of little intelligence is destroyed.
```

File: _content/excerpt/mulamadhymakakarika_garfield-jay.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---
+title: "Mulamadhymakakarika"
+translator: garfield-jay
+external_url: ""
+drive_links:
+  - "https://drive.google.com/file/d/187T2v8GMkuY7rT6Rl7a_oeXXSWKLn7u3/view?usp=drive_link"
+formats: [pdf]
+course: view
+tags:
```
Comment (buddhist-uni) [inline _content/excerpt/mulamadhymakakarika_garfield-jay.md:9]: good tags. Let's also add `sects` perhaps? And `status: featured`! 😁

File: _content/excerpt/mulamadhymakakarika_garfield-jay.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  - nagarjuna
+  - madhyamaka
+  - mahayana
+  - emptiness
+year: 1995 
+oclc: 32012411 
+publisher: oxford
+address: "United Kingdom"
```
Comment (buddhist-uni) [inline _content/excerpt/mulamadhymakakarika_garfield-jay.md:17]: Yeah, UK is ok... or "Oxford"...

**Review by buddhist-uni — COMMENTED** [review]
Yes! A taste of Madhyamaka for the Right View course! 🤭

**Comment by JJM317** [comment]
I'll get around to fixing these today or tomorrow. I wanted to make sure
you saw my questions in the comments box when I created the pull request.

Kindly,
~Joe
--------------------

File: _content/excerpt/selected-verses-mulamadhymakakarika_garfield-jay.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+  - sects  
+editor: "Rev. Yin Shun"
+year: 1995 
+publisher: oxford
+address: "Oxford"
+booktitle: "The fundamental wisdom of the middle way: Nāgārjuna's Mūlamadhyamakakārikā"
+pages: 12
+---
+> By a misperception of emptiness a person of little intelligence is destroyed.
+A translation of a select seventy verses from Nāgārjuna's Mūlamadhyamakakārikā.
```
Comment (buddhist-uni) [inline _content/excerpt/selected-verses-mulamadhymakakarika_garfield-jay.md:24]: FYI: there needs to be a blank line between the pull quote and your comments

**Comment by buddhist-uni** [comment]
oops! You put this in `excerpt` not `excerpts`  All the category folder names are plural, for future reference 😊

**Pull request merged**